package.json

```diff
@@ -31,5 +31,9 @@
 		"rollup": "^3.14.0",
 		"sass": "^1.58.0",
 		"typescript": "^4.9.5"
+	},
+	"dependencies": {
```

Comment:
Should be under `dev-dependencies`

Reply:
bulma-toast needs to move, although the JWT library should stay. That library is needed during upload.